AppCenter/AppCenterTests/MSUtilityTests.m

```diff
@@ -897,6 +897,28 @@ - (void)testObfuscateRedirectUri {
   XCTAssertFalse([obfuscatedString rangeOfString:kMSRedirectUriObfuscatedTemplate].location == NSNotFound);
 }
 
+- (void)testDispatchObjectMacro {
+
+  // If
+  NSMutableArray *array = [NSMutableArray new];
+
+  // When
+  MS_DISPATCH_SELECTOR((void (*)(id, SEL, id)), array, addObject:, @"test");
+
+  // Then
+  XCTAssertEqual([array count], 1);
+  XCTAssertEqual([array firstObject], @"test");
+}
+
+- (void)testDispatchObjectMacroWithNil {
+  typedef void (^block)(NSString *, NSString *);
```

Comment:
Smart test case 👍

Reply:
We could also have used a mock to validate the call with specific arguments but this works too.